### DISCUSSION TITLE
Ran /uploadssc and my SSC character disappeared, where did it go?

### DISCUSSION THREAD

**Discussion by BrianNether** [Pryaxis/TShock#2037] [Q&A]
Trying to mess with SSC files, I ran /uploadssc to see what it did. All my character data disappeared and was replaced with the default copper tools. What did /uploadssc do and how can I recover my character data?

**Comment by hakusaro** [comment]
It should have uploaded your existing data to the server as the SSC data for your user. If your local character data got deleted and the server doesn't have it, I don't have a good solution for you. You can try examining the tshock database directly, but that's about all I can suggest.
